.github/workflows/publish_docs.yaml

```diff
@@ -33,7 +28,7 @@ jobs:
         run: pip install -r docs/fides/requirements.txt
 
       - name: Install fidesctl
-        run: pip install -e fidesctl/
+        run: pip install -e fidesctl/[all]
```

Comment:
does this need quotes to work?

`pip install -e "fidesctl/[all]"`

Reply:
Nope!